patches/chrome-browser-resources-settings-site_settings-category_default_setting.ts.patch

```diff
@@ -6,7 +6,7 @@ index 306b03008c00e43d7cea89ec0af003c488e37f71..d0a0d9b7d37eb8b36653dcf052ababc5
       */
      switch (this.category) {
        case ContentSettingsTypes.ADS:
-+      case ContentSettingsTypes.AUTOPLAY:
++      case ContentSettingsTypes.AUTOPLAY: case ContentSettingsTypes.SHIELDS: case ContentSettingsTypes.HTTP_UPGRADABLE_RESOURCES:
```

Comment:
@petemill is there a way to either remove these patches or make them extensible?

Reply:
It's possible. I just noticed we now also patch this same function further down for the "ask" types.

We would do this:
```js
// brave_overrides/category_default_setting.js

import {RegisterPolymerComponentReplacement} from 'chrome://brave-resources/polymer_overriding.js'
import {CategoryDefaultSettingElement} from '../site_settings/category_default_setting.js'

RegisterPolymerComponentReplacement(
  'category-default-setting',
  class BraveCategoryDefaultSettingElement extends
      CategoryDefaultSettingElement {
    onChangePermissionControl_() {
      // Handle our own enum values here
      super.onChangePermissionControl_()
    }
  }
)
```

but I cannot actually get this control to render to test it out - do we actually need this patch?

Reply:
it is not needed anymore because we decided to remove default categories, removed patch